packages/nimble-blazor/NimbleBlazor/Components/NimbleTabs.razor.cs

```diff
@@ -16,13 +16,20 @@ public partial class NimbleTabs : ComponentBase
     [Parameter] public EventCallback<string?> ActiveIdChanged { get; set; }
 
     /// <summary>
-    /// Called when activeid changes on the web component
+    /// Called when `change` event is called on the web component or its children
     /// </summary>
+    /// <remarks>
+    /// If <paramref name="value"/> is null, ignore the change event. Due to the way
+    /// Blazor Custom Events work, this event gets triggered when a child <see cref="NimbleTextField"/>'s Value changes.
```

Comment:
Made it private, deleted the documentation, and changed it from a path to a minor (since it is a slight API change).  I don't think it is breaking to clients or else I'd have bumped it up even higher.